src/main/java/cz/cvut/kbss/termit/rest/AdminBasedRegistrationController.java

```diff
@@ -56,11 +60,17 @@ public AdminBasedRegistrationController(UserService userService) {
         LOG.debug("Instantiating admin-based registration controller.");
     }
 
+    @Operation(security = {@SecurityRequirement(name="bearer-key")},
+               description = "Creates a new user account. If the password is blank, the account is locked, and an email will be sent to the new user with a link to create a password.")
+    @ApiResponses({
+            @ApiResponse(responseCode = "201", description = "User created"),
+            @ApiResponse(responseCode = "409", description = "User data are invalid")
+    })
     @PreAuthorize("hasRole('" + SecurityConstants.ROLE_ADMIN + "')")
-    @PostMapping(consumes = {MediaType.APPLICATION_JSON_VALUE, JsonLd.MEDIA_TYPE})
+    @PutMapping(consumes = {MediaType.APPLICATION_JSON_VALUE, JsonLd.MEDIA_TYPE})
```

Comment:
Currently, `AdminBasedRegistrationController` is mapped to the same path as [`FreeRegistrationController`](https://github.com/kbss-cvut/termit/blob/da05402700f6ad7e7937e2bd46cd12657fe8c180/src/main/java/cz/cvut/kbss/termit/rest/FreeRegistrationController.java#L50) and `FreeRegistrationController.createUser` shared the same mapping with `AdminBasedRegistrationController.createUser`, wasn't a problem because only one was always active (based on profile `admin-registration-only`)  
But since I set `AdminBasedRegistrationController` as always active, I had to remove the mapping conflict.  
Another option is to change the mapping path to something like `/admin/users`.

Reply:
Ok, in that case I vote to change the path mapping to `/admin/users`